### PR TITLE
feat: 🎸 implement new ca events

### PR DESCRIPTION
**Pull request by sansan**
### Description

updates to use new event names for CA

### Breaking Changes

Requires resync 

### JIRA Link

https://polymesh.atlassian.net/browse/DA-1090

### Checklist

- [x] Updated the Readme.md (if required) ?

### CODE REVIEW

**Review by polymath-eric — APPROVED** [review]
lgtm

**Review by prashantasdeveloper — APPROVED** [review]

File: schema.graphql [hunk 1]
```diff
@@ -2422,6 +2434,7 @@ type ConfidentialAsset @entity {
   totalSupply: BigInt!
   allowedVenues: [Int!]
   venueFiltering: Boolean!
+  isFrozen: Boolean
```
Comment (prashantasdeveloper) [inline schema.graphql:2437]:
```suggestion
  isFrozen: Boolean!
```
by default set this value to false while handling `AssetCreated` event

**Review by prashantasdeveloper — CHANGES_REQUESTED** [review]
ConfidentialAssets should by default set

**Comment by prashantasdeveloper** [comment]
@sansan I think you forgot to push changes and requested a re-review w/o updating the entity

**Pull request merged**